apps/docs/src/examples/inputs/TagsDropdown.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useRef, useState } from "react";
+import {
+  HvBaseDropdown,
+  HvLabel,
+  HvPanel,
+  HvTag,
+} from "@hitachivantara/uikit-react-core";
+
+const colors = [
+  "Blue",
+  "Red",
+  "Green",
+  "Yellow",
+  "Purple",
+  "White",
+  "Black",
+  "Orange",
+  "Pink",
+  "Brown",
+  "Gray",
+  "Cyan",
+  "Magenta",
+  "Lime",
+  "Teal",
+  "Lavender",
+];
+
+export default function Demo() {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const focusTarget = useRef<HTMLDivElement>(null);
+
+  const [selectedColors, setSelectedColors] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.scrollLeft = containerRef.current.scrollWidth;
+    }
+  }, [selectedColors]);
```

Comment:
we own `selectedColors`, so this `useEffect` could easily be avoided by moving the code to where we do `setSelectedColors()`